transactions/2.0/_index.md

```diff
@@ -5,19 +5,19 @@ summary: "Release for Jakarta EE 9"
 ---
 Jakarta Transactions - Need scope statement
 
-* [Jakarta Transactions 2.0 Release Record]()
+* [Jakarta Transactions 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jta/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
 * [Jakarta Transactions 2.0 Specification Document]() (PDF)
 * [Jakarta Transactions 2.0 Specification Document]() (HTML)
 * [Jakarta Transactions 2.0 Javadoc](./apidocs)
```

Comment:
Putting the apidocs in the same PR is just fine.  We have been discussing the need to simplify this process and condensing down to a single PR is one of the items.  Thanks.

Reply:
Thanks for the confirmation!